### ISSUE TITLE
Some inputs are missing programmatic labels

### ISSUE THREAD

**Issue by connerbw** [johnbillion/extended-cpts#119]
During a recent accessibility audit, we were informed that some of our inputs are missing programmatic labels. 

A few of them come from this plugin. Specifically from `wp_dropdown_categories`

When the `<select` inputs are rendered there should be an associated `<label for="x">`, or an `aria-label`, for them.

**Comment by johnbillion** [comment]
Thanks for the report!
